src/pages/workspace/categories/WorkspaceCategoriesPage.tsx

```diff
@@ -62,38 +62,49 @@ function WorkspaceCategoriesPage({policyCategories, route}: WorkspaceCategoriesP
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
-    const categoryList = useMemo<PolicyForList[]>(
+    const categoryList = useMemo(
```

Comment:
```suggestion
    const categoryList = useMemo<PolicyOption>(
```

Reply:
Is this necessary? We already have the PolicyOption type specified in the map function, which means the type of the category list is already defined.

Reply:
Yeah, I saw it. I think that the typing of the `useMemo` is preferable. I would say it is better to type a kind of high-level function.